sw/device/tests/BUILD

```diff
@@ -4384,6 +4384,36 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "sysrst_ctrl_ec_rst_l_test",
+    srcs = ["sysrst_ctrl_ec_rst_l_test.c"],
+    cw310 = new_cw310_params(
+        test_cmd = """
+            --bitstream="{bitstream}"
+            --bootstrap="{firmware}"
+        """,
+        test_harness = "//sw/host/tests/chip/sysrst_ctrl:sysrst_ctrl_ec_rst_l",
+    ),
+    exec_env = {
+        "//hw/top_earlgrey:fpga_cw310_sival": None,
```

Comment:
Should the silicon targets be added?

Reply:
I will add the silicon targets after I am finished with all the sysrst_ctrl tests.